code/__DEFINES/mode.dm

```diff
@@ -112,6 +112,8 @@
 #define COOLDOWN_COMM_MESSAGE 	1 MINUTES
 #define COOLDOWN_COMM_CENTRAL 	30 SECONDS
 
+#define SUPPLY_POINT_MARINE_SPAWN 5
```

Comment:
```suggestion
#define SUPPLY_POINT_MARINE_SPAWN 5
```
Actually i believe this should be gamemode dependant, just add a var to gamemode type for this

Reply:
The only gamemode that uses req point is distress, no crash...?